src/components/Modal/components/DeleteArticle.tsx

```diff
@@ -0,0 +1,17 @@
+import ModalContainer from '../ModalContainer';
+
+const DeleteArticle = () => {
+  return (
+    <ModalContainer
+      title="해당 글을 삭제할까요?"
+      secondButtonProps={{
+        text: '삭제하기',
+        onClick: () => {},
+      }}
+    >
+      {'삭제한 글은 복구할 수 없어요!\n삭제하시겠어요'}
+    </ModalContainer>
+  );
+};
```

Comment:
```suggestion
const DeleteArticle = () => {
  return (
    <ModalContainer>
      <ModalHeader>
        <h1>해당 글을 삭제할까요?</h1>
      </ModalHeader>
      <ModalContent>
        <p>{'삭제한 글은 복구할 수 없어요!\n삭제하시겠어요'}</p>
      </ModalContent>
      <ModalFooter>
        <button>취소</button>
        <button>삭제하기</button>
      </ModalFooter>
    </ModalContainer>
  );
};
```
추상화가 된다면 이렇게 활용할 수 있습니다!